src/objects.rs

```diff
@@ -634,6 +637,41 @@ impl Frequency {
     }
 }
 
+/// Transfer information between stops before merged into [Stop]
+#[derive(Debug, Serialize, Deserialize, Default)]
+pub struct RawTransfer {
+    /// Stop from which to leave
+    pub from_stop_id: String,
+    /// Stop which to transfer to
+    pub to_stop_id: String,
+    /// Type of the transfer
+    pub transfer_type: TransferType,
+    /// Minimum time needed to make the transfer in seconds
+    pub min_transfer_time: Option<u32>,
+}
+
+#[derive(Debug, Default, Clone)]
+/// Transfer information between stops
+pub struct StopTransfer {
+    /// Stop which to transfer to
+    pub to_stop_id: String,
+    /// Type of the transfer
+    pub transfer_type: TransferType,
+    /// Minimum time needed to make the transfer in seconds
+    pub min_transfer_time: Option<u32>,
+}
+
+impl StopTransfer {
+    /// Converts from a [RawTransfer] to a [StopTransfer]
+    pub fn from(transfer: RawTransfer) -> Self {
```

Comment:
I’m nitpicking here ;)
```suggestion
impl From<RawTransfer> for StopTransfer {
    /// Converts from a [RawTransfer] to a [StopTransfer]
    fn from(transfer: RawTransfer) -> Self {
```